conf/jvm17-server.options

```diff
@@ -87,7 +87,7 @@
 # Java 11 (and newer) GC logging options:
 # See description of https://bugs.openjdk.java.net/browse/JDK-8046148 for details about the syntax
 # The following is the equivalent to -XX:+PrintGCDetails -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=10M
--Xlog:gc=info,heap*=debug,age*=debug,safepoint=info,promotion*=debug:file=/var/log/cassandra/gc.log:time,uptime,pid,tid,level:filecount=10,filesize=10485760
+# -Xlog:gc=info,heap*=debug,age*=debug,safepoint=info,promotion*=debug:file=/var/log/cassandra/gc.log:time,uptime,pid,tid,level:filecount=10,filesize=10485760
```

Comment:
Could you maybe prepare `jvm22-server.options` and `jvm22-client.options`? I think the client options will be useful when we eventually run Python DTests with j22

Reply:
Yes, I will. This was hacky branch for testing/experimenting with the unit tests. I will wrap up something representative today